homeassistant/components/cover/tilt.py

```diff
@@ -0,0 +1,156 @@
+"""
+Tilt Cover
+Cover component for Home Assistant https://home-assistant.io/
+Documentation: https://home-assistant.io/components/cover.tilt/
+Jerry Workman <jerry.workman@gmail.com>
+"""
+
+
+from threading import Timer
+from homeassistant.components.cover import CoverDevice
+from homeassistant.components import switch
+from homeassistant.const import (STATE_OPEN, STATE_CLOSED, STATE_ON, STATE_OFF)
+from homeassistant.components.cover import PLATFORM_SCHEMA
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+import logging
+import pprint
```

Comment:
This is not needed as outputting the configuration in line 40 should be avoided.